deepchem/utils/protein_sequence_data_utils.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Oct 12 10:01:57 2020
+
+@author: Zhenqin Wu
+"""
+import os
+import numpy as np
+
+def read_fasta(path):
```

Comment:
It would be very useful to get docstrings in for these utilities and type signatures as well :)